subprojects/snapshots/src/main/java/org/gradle/internal/fingerprint/FingerprintingStrategy.java

```diff
@@ -42,7 +42,6 @@
     FingerprintHashingStrategy getHashingStrategy();
 
     /**
-     * UsedByScansPlugin
      * Names are expected as part of org.gradle.api.internal.tasks.SnapshotTaskInputsBuildOperationType.Result.VisitState.getPropertyNormalizationStrategyName().
```

Comment:
I think the whole Javadoc can go away now, right?

Reply:
Indeed, as we don't use it to generate the attributes anymore.